src/swap/SwapPending.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useEffect, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ActivityIndicator, BackHandler, StyleSheet, Text, View } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { useDispatch, useSelector } from 'react-redux'
+import Button from 'src/components/Button'
+import Dialog from 'src/components/Dialog'
+import Checkmark from 'src/icons/Checkmark'
+import { noHeader } from 'src/navigator/Headers'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { swapStateSelector } from 'src/swap/selectors'
+import { swapReset, SwapState } from 'src/swap/slice'
+
+export function SwapPending() {
+  const swapState = useSelector(swapStateSelector)
+  const dispatch = useDispatch()
+  const { t } = useTranslation()
+
+  const navigateToSwapStart = () => {
+    dispatch(swapReset())
+    navigate(Screens.SwapScreen)
+  }
+
+  const navigateToReviewScreen = () => {
+    navigate(Screens.SwapReviewScreen)
+  }
+
+  const navigateToSupport = () => {
+    dispatch(swapReset())
+    navigate(Screens.SwapScreen)
+    navigate(Screens.SupportContact)
```

Comment:
ah, i forgot i was going to help look into this double navigate...i'll put it on my list

Reply:
i'm thinking that this issue isn't worth spending too much time on, but i think navigating twice is quite hacky (for lower end devices like mine, this double navigation can actually be seen). can i suggest for MVP that we just use the `replace` function in the navigation service so that when you navigate back from support, you land on the swap review screen? that seems reasonable, to see that swap that you've contacted support for?